### DISCUSSION TITLE
How to use icons in tab titles?

### DISCUSSION THREAD

**Discussion by thedevilfish** [squidfunk/mkdocs-material#2107] [Q&A] [answered]
I was trying to see if you can put an icon on the tab itself but it seems like it doesn't render correctly. Anyone know if this is supported?

```md
=== "Mac OS :fontawesome-brands-apple:"
        These are instructions for Mac OS
=== "Linux :fontawesome-brands-linux:"
        These are instructions for Linux
```

Any thoughts?

**Comment by facelessuser** [comment]
This is something already planned in the pymdown-extensions library that implements the tabs.

**Comment by facelessuser** [comment]
In the next release, I will relax titles so they can be parsed by Markdown the same way Admonitions and Details does. While I'd like the syntax logic to be reworked to be a  little more sane in the future, I think it makes sense to allow Tabbed to work like the other similar extensions which 99% of the time works how people expect them to.

<img width="207" alt="Screen Shot 2020-12-24 at 10 23 46 AM" src="https://user-images.githubusercontent.com/1055125/103100915-2d46de80-45d2-11eb-924e-abe1c2b87b53.png">

**Comment by facelessuser** [comment, marked as answer]
Logic in Tabbed to allow for emoji and such as been released in 8.1.0.
